source/ClientApp/src/app/events/Models/event.model.ts

```diff
@@ -0,0 +1,10 @@
+export interface Event {
+
+  eventId: any;
+  organizerId: string;
+  guestListId: any;
```

Comment:
This is correct.  I have the eventId here so that when the object is returned from the db, the eventId will be available.  Is there a better way to handle this in Angular so it's not modifiable?